libnetwork/drivers/bridge/interface_linux_test.go

```diff
@@ -16,35 +44,142 @@ func TestInterfaceDefaultName(t *testing.T) {
 	}
 	config := &networkConfiguration{}
 	_, err = newInterface(nh, config)
-	if err != nil {
-		t.Fatalf("newInterface() failed: %v", err)
-	}
+	assert.NilError(t, err)
+	assert.Equal(t, config.BridgeName, DefaultBridgeName)
```

Comment:
We _might_ be able to always test both here, even on error (getting more details on failures (where we can) never hurts);

```go
assert.Check(t, err)
assert.Check(t, is.Equal(config.BridgeName, DefaultBridgeName))
```

Reply:
Yes, will do.